smk/assembly.smk

```diff
@@ -247,7 +247,7 @@ def get_runs_for_sample(wildcards):
             omics=wildcards.omics,
             location=get_qc2_output_location(wildcards.omics),
             illumina=wildcards.illumina)
-    runs = [ re.sub("\.1\.fq\.gz", "", path.basename(f)) for f in os.scandir(sample_dir) if f.is_file() and f.name.endswith('.1.fq.gz') ]
+    runs = [ re.sub(r"\.1\.fq\.gz", "", path.basename(f)) for f in os.scandir(sample_dir) if f.is_file() and f.name.endswith('.1.fq.gz') ]
```

Comment:
My understanding of r"" is that it's mean to be used to preserve back-slashes for printing in cases when they could also be interpreted as "\n" or "\s" characters. If patterns are expected fx. in re.sub(), I don't believe they need to be used.
Also not necessary for use with triple-quotes.

Reply:
It works without r"" in our python but printed warning messages in Titus' ubuntu servers for the tutorial. And warnings disappeared after adding r"". Also for triple-quotes where the perl command uses lots of `\d` failed on the tutorial servers. Only worked after adding r""". In a way, we are porting to newer versions of python with this...